### PR TITLE
Add Feishu provider

### PR DESCRIPTION
**Pull request by vicenteyu**
Add Feishu provider.

### CODE REVIEW

File: src/AspNet.Security.OAuth.Feishu/AspNet.Security.OAuth.Feishu.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```
Comment (martincostello) [inline src/AspNet.Security.OAuth.Feishu/AspNet.Security.OAuth.Feishu.csproj:6]:
```suggestion
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
    <PackageValidationBaselineVersion>6.0.9</PackageValidationBaselineVersion>
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```

**Review by martincostello — COMMENTED** [review]
This seems fine. Could you add the new provider to the table in the README at the root of the repository too please?

**Comment by martincostello** [comment]
Oh, and it needs adding to the solution file too.

**Review by martincostello — APPROVED** [review]

**Comment by martincostello** [comment]
Just to confirm, have you tested that this code works correctly against the real Feishu service using a sample application?

**Comment by vicenteyu** [comment]
> Just to confirm, have you tested that this code works correctly against the real Feishu service using a sample application?

Yes. It works fine.

QQ截图20220821003422
QQ截图20220821003654

**Comment by martincostello** [comment]
Thanks again for your contribution. This package is now available from [NuGet.org](https://www.nuget.org/packages/AspNet.Security.OAuth.Feishu/).

**Pull request merged**